src/formatters/otp_formatter.erl

```diff
@@ -1194,12 +1190,23 @@ lay_clause_expressions([H | T], Ctxt) ->
     end;
 lay_clause_expressions([], _) -> empty().
 
+is_last_and_before_empty_line(H, [], #ctxt{empty_lines = EmptyLines}) ->
```

Comment:
This block is what's making `otp_formatter_SUITE` fail at the moment.

Reply:
Yeah. I _guess_ it's because of the additional line at the end of the file… but I'm not 100% sure.

Reply:
This comment makes no sense anymore.